test/Symbols.test.js

```diff
@@ -0,0 +1,40 @@
+const assert = require('assert');
+const math = require('mathjs');
+
+const print = require('../lib/util/print');
+const Symbols = require('../lib/Symbols');
+
+function runTest(functionToTest, exprString, expectedOutput) {
+  it(exprString + ' -> ' + expectedOutput, function () {
+    const expression = math.parse(exprString);
+    const foundSymbol = functionToTest(expression, 'x');
+    assert.deepEqual(
+      print(foundSymbol),
+      expectedOutput
+    );
+  });
+}
+
+describe('getLastSymbolTerm', function() {
+  const tests = [
+    ['1/x', '1 / x'],
+    ['1/(3x)', '1 / (3x)'],
+    ['3x', '3x'],
+    ['x + 3x', '3x'],
+    ['x/(x+3)', 'x / (x + 3)'],
+  ];
+
+  tests.forEach(t => runTest(Symbols.getLastSymbolTerm, t[0], t[1]));
+});
+
+describe('getLastDenominatorWithSymbolTerm', function() {
+  const tests = [
+    ['1/x', 'x'],
+    ['1/(x+2)', '(x + 2)'],
+    ['1/(x+2) + 3x', '(x + 2)'],
+    ['1/(x+2) + 3x/(1+x)', '(1 + x)'],
```

Comment:
Done.  There was even an existing issue with how symbols were being parsed that was made obvious with this particular test case.  I've added a commit to recursively parse the nodes for symbols.  It was previously only comparing `2 + 2/x` and `x/2` as the only two nodes, instead of checking `2` and `2/x` individually